scripts/start_local.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Comment:
I'm not certain, but I think the reason this was originally below was so a more helpful message was returned in the case of failure.

For example, if I set a dummy profile that doesn't exist with the new credential processing code:

```bash
export AWS_PROFILE=blah
```

With `set -e` at the top:

```bash
make up
bash scripts/start_local.sh
make: *** [up] Error 1
```

With `set -e` in the original location:

```sh
make up
bash scripts/start_local.sh

The config profile (blah) could not be found
Error: AWS_ACCESS_KEY_ID was not set and could not be loaded from AWS_PROFILE
make: *** [up] Error 1
```

I'm okay with moving it to the top, but I think there needs to be additional handling in the new code

Reply:
I moved back to original location for now.